### PR TITLE
:hammer: API_Player setMarkerPosition

### PR DESCRIPTION
**Pull request by Zeleeer**
easy to use : 

```NCS.Player:setMarkerPosition()```

### CODE REVIEW

File: src/api/player/client/setMarkerPosition.lua [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---setMarkerPosition
+---@return boolean
+---@public
+function API_Player:setMarkerPosition()
+    local blipPos = GetFirstBlipInfoId(8)
+    if not (DoesBlipExist(blipPos)) then
+        return false
```
Comment (MoskalykA) [inline src/api/player/client/setMarkerPosition.lua:7]: return NCS:die(...)

File: src/api/player/client/setMarkerPosition.lua [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---setMarkerPosition
+---@return boolean
```
Comment (MoskalykA) [inline src/api/player/client/setMarkerPosition.lua:2]:
```suggestion
```

File: src/api/player/client/setMarkerPosition.lua [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+            zCoords = zCoords + 10.0
+            RequestCollisionAtCoord(blipCoords.x, blipCoords.y, zCoords)
+            Wait(0)
+            foundGround, zPos = GetGroundZFor_3dCoord(blipCoords.x, blipCoords.y, zCoords)
+            if not (foundGround) and (zCoords >= 2000.0) then
+                foundGround = true
+            end
+        end
+        SetPedCoordsKeepVehicle(PlayerPedId(), blipCoords.x, blipCoords.y, zPos)
+        return true
```
Comment (MoskalykA) [inline src/api/player/client/setMarkerPosition.lua:21]:
```suggestion
```

File: src/api/player/client/setMarkerPosition.lua [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+        local foundGround, zCoords, zPos = false, -500.0, 0.0
+        while (not foundGround) do
+            zCoords = zCoords + 10.0
+            RequestCollisionAtCoord(blipCoords.x, blipCoords.y, zCoords)
+            Wait(0)
+            foundGround, zPos = GetGroundZFor_3dCoord(blipCoords.x, blipCoords.y, zCoords)
+            if not (foundGround) and (zCoords >= 2000.0) then
+                foundGround = true
+            end
+        end
```
Comment (MoskalykA) [inline src/api/player/client/setMarkerPosition.lua:19]: What is the meaning of this piece of code?

**Review by MoskalykA — CHANGES_REQUESTED** [review]

File: src/api/player/client/setMarkerPosition.lua (hunk 4 above)
Comment (Zeleeer) [reply src/api/player/client/setMarkerPosition.lua:19]: is request Z coords, so as not to going under the map

**Review by MoskalykA — APPROVED** [review]

**Pull request merged**